addons/website/views/website_templates.xml

```diff
@@ -965,21 +969,48 @@
     </div>
 </template>
 
+<template id="400">
+    <t t-call="website.layout">
+        <div id="wrap">            
+            <t t-if="request.debug">
+                <div class="container">
+                    <h1 class="mt32">Oops! Something went wrong.</h1>
+                </div>
+                <t t-call="website.http_error_debug"/>
+            </t>
+            <t t-else="">
+                <div class="container">
+                    <h1 class="mt32">Oops! Something went wrong.</h1>
+                    <p>Take a look at the error message below.</p>
+                    <t t-call="website.error_message"/>
+                </div>
+            </t>
+        </div>
+    </t>
+</template>
+
 <template id="403">
     <t t-call="website.layout">
         <div id="wrap">
-            <div class="container">
-                <h1 class="mt32">403: Forbidden</h1>
-                <p>The page you were looking for could not be authorized.</p>
-                <p>Maybe you were looking for one of these popular pages ?</p>
-                <ul>
-                    <li><a href="/">Homepage</a></li>
-                    <li><a href="/contactus">Contact Us</a></li>
-                </ul>
-            </div>
-            <t t-if="editable or request.debug">
+            <t t-if="request.debug">
+                <div class="container">
+                    <h1 class="mt32">403: Forbidden</h1>
+                    <p>The page you were looking for could not be authorized.</p>
+                </div>
                 <t t-call="website.http_error_debug"/>
             </t>
+            <t t-else="">
+                <div class="container">
+                    <h1 class="mt32">403: Forbidden</h1>
+                    <p>The page you were looking for could not be authorized.</p>
+                    <t t-call="website.error_message"/>
+                    <p>Maybe you were looking for one of these popular pages ?</p>
+                    <ul>
+                        <li><a href="/">Homepage</a></li>
```

Comment:
If we think we need to add links on this error page, why not in other errors too?